chore: add note about GitHub App allowed IP addresses (#3048)
Mergifyio/docs: src/content/security.mdx

default](/commands/restrictions/#default-restrictions) and have their own
mechanism that can be modified. See [Commands
Restrictions](/commands/restrictions/) for changing the default.

> ## Managing IP addresses allowed for our GitHub App

Reviewer: ```suggestion
## Managing IP Addresses Allowed for the GitHub App
```

> [GitHub allows to configure the list of IP adresses](https://docs.github.com/en/apps/maintaining-github-apps/managing-allowed-ip-addresses-for-a-github-app
> ) that a GitHub App can use to reach GitHub.

Reviewer: ```suggestion
[GitHub allows to configure the list of IP adresses](https://docs.github.com/en/apps/maintaining-github-apps/managing-allowed-ip-addresses-for-a-github-app
) that a GitHub App is allowed to use to access GitHub.
```

> Mergify services are hosted on Heroku us region and does not provide static IP addresses. You must allow
> the AWS EC2 us-east-1 IP addresses.
> 

Reviewer: ```suggestion
Mergify services do not use a single static IP address. Therefore, you must allow the AWS EC2 `us-east-1` IP addresses.

```
